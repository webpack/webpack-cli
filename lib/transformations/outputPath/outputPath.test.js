const defineTest = require('../defineTest');

defineTest(__dirname, 'outputPath', 'outputPath-0');
defineTest(__dirname, 'outputPath', 'outputPath-1');
defineTest(__dirname, 'outputPath', 'outputPath-2');
