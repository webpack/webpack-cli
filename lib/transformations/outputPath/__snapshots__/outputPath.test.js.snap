// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputPath transforms correctly using "outputPath-0" data 1`] = `
"module.exports = {
	output: {
		path: path.join(__dirname, 'dist')
	}
}
"
`;

exports[`outputPath transforms correctly using "outputPath-1" data 1`] = `
"const path = require('path');
module.exports = {
	output: {
		path: path.join(__dirname, 'dist')
	}
}
"
`;

exports[`outputPath transforms correctly using "outputPath-2" data 1`] = `
"const p = require('path');
module.exports = {
	output: {
		path: p.join(__dirname, 'dist')
	}
}
"
`;
